Content.Shared/Nutrition/Components/HungerComponent.cs

```diff
@@ -61,6 +61,14 @@ public sealed partial class HungerComponent : Component
     [AutoNetworkedField]
     public HungerThreshold CurrentThreshold;
 
+    /// <summary>
+    /// Goobstation
+    /// Starting hunger value the entity should be at, if set then it overrides the default hunger value randomization behaviour.
+    /// </summary>
+    [DataField("startingHunger"), ViewVariables(VVAccess.ReadOnly)]
+    [AutoNetworkedField]
```

Comment:
```suggestion
    [DataField, ViewVariables(VVAccess.ReadOnly)]
```
Remove explicit tag from datafield. Also this variable is meant to be changed only in yaml so no networking needed.

Reply:
:caught: I blame thirstcomponent having the same one.